src/main/scss/templates/_messages.scss

```diff
@@ -0,0 +1,104 @@
+// Messages styling
+.messages {
+  display: flex;
+  padding: $size-base * 2;
+  height: 100%;
+
+  .card {
```

Comment:
Maybe we should throw this into the root .card style. Seems like we're only ever using it with these specifications (bg color, border radius, margin, etc). Up to you to do in this PR or add a TODO/issue.